src/main/kotlin/com/vk/admstorm/startup/ChangeSshBackendStartup.kt

```diff
@@ -10,44 +10,50 @@ import com.vk.admstorm.notifications.AdmNotification
 object ChangeSshBackendStartup {
     private val LOG = logger<ChangeSshBackendStartup>()
 
-    private const val SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
+    private const val LEGACY_SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
     private const val SSH_CONFIG_BACKEND = "ssh.config.backend"
+    private const val SSH_NOTIFICATOR_OPTION = "sshOpenSSHChanges"
```

Comment:
Let's move it higher to be next to `LEGACY_SSH_NOTIFICATOR_OPTION`

Reply:
Fix